src/Message/AbstractOrderRequest.php

```diff
@@ -172,16 +174,22 @@ public function setWidgetOptions($widgetOptions)
 
     /**
      * @return array
+     *              
+     * @throws InvalidRequestException
      */
     protected function getOrderData()
     {
-        $data = [
-            'order_amount' => $this->getAmountInteger(),
-            'order_tax_amount' => $this->toCurrencyMinorUnits($this->getTaxAmount()),
-            'order_lines' => $this->getItemData($this->getItems()),
-            'purchase_currency' => $this->getCurrency(),
-            'purchase_country' => $this->getPurchaseCountry(),
-        ];
+        try {
+            $data = [
+                'order_amount' => $this->getAmountInteger(),
+                'order_tax_amount' => $this->toCurrencyMinorUnits((string) $this->getTaxAmount()),
+                'order_lines' => $this->getItemData($this->getItems()),
+                'purchase_currency' => $this->getCurrency(),
+                'purchase_country' => $this->getPurchaseCountry(),
+            ];
+        } catch (UnknownCurrencyException $exception) {
```

Comment:
Where does this exception originate, somewhere in `toCurrencyMinorUnits`?

Reply:
IDK will remove it!